EssentialsDiscordLink/src/main/java/net/essentialsx/discordlink/EssentialsDiscordLink.java

```diff
@@ -0,0 +1,108 @@
+package net.essentialsx.discordlink;
+
+import com.earth2me.essentials.IEssentials;
+import com.earth2me.essentials.metrics.MetricsWrapper;
+import com.google.common.collect.ImmutableSet;
+import net.essentialsx.api.v2.services.discord.DiscordService;
+import net.essentialsx.api.v2.services.discord.InteractionException;
+import net.essentialsx.discord.EssentialsDiscord;
+import net.essentialsx.discordlink.commands.discord.AccountInteractionCommand;
+import net.essentialsx.discordlink.commands.discord.LinkInteractionCommand;
+import net.essentialsx.discordlink.commands.discord.UnlinkInteractionCommand;
+import net.essentialsx.discordlink.listeners.LinkBukkitListener;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.plugin.java.JavaPlugin;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import static com.earth2me.essentials.I18n.tl;
+
+public class EssentialsDiscordLink extends JavaPlugin {
+    private final static Logger logger = Logger.getLogger("EssentialsDiscordLink");
+    private transient IEssentials ess;
+    private transient EssentialsDiscord essDiscord;
+    private transient MetricsWrapper metrics = null;
+
+    private DiscordService api;
+    private DiscordLinkSettings settings;
+    private AccountStorage accounts;
+    private AccountLinkManager linkManager;
+
+    @Override
+    public void onEnable() {
+        ess = (IEssentials) getServer().getPluginManager().getPlugin("Essentials");
+        essDiscord = (EssentialsDiscord) getServer().getPluginManager().getPlugin("EssentialsDiscord");
+        if (ess == null || !ess.isEnabled() || essDiscord == null || !essDiscord.isEnabled()) {
+            setEnabled(false);
```

Comment:
Probably want to log a message about requiring EssentialsX Discord to run

Reply:
This is dead code anyway, EssentialsDiscord is in the `depend` in `plugin.yml`